test/system/roles/public_health/dashboard/export_verifier.rb

```diff
@@ -16,26 +16,20 @@ def verify_csv_linelist(user_label, workflow)
     user = @@system_test_utils.get_user(user_label)
     export_type = "csv_linelist_#{workflow}".to_sym
     download_export_files(user, export_type)
-    patients = user.jurisdiction.all_patients_excluding_purged.where(isolation: workflow == :isolation)
+    patients_group = user.jurisdiction.all_patients_excluding_purged.where(isolation: workflow == :isolation).reorder('')
```

Comment:
I think we should keep this as `patients` and rename `patients_group` on line 22 to `patients` instead since these patients are no longer a result of batching